docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -69,6 +69,20 @@
 * Updated the Repository and Cart to display Sample Sheet and Metadata downloads as separate buttons for better visibility. <!--PEAR-2232-->
 * Minor text and styling improvements. <!--PEAR-452/2298/2341/2323/2253/2290-->
 
+* __Single-Cell RNAseq__:
```

Comment:
```suggestion
* __New Analysis Tool:  Single-Cell RNAseq__:
```

I suggest putting this at the very top as analysis tools are a big part of the Data Portal.  Also add a return under "### New Features and Changes" (just to save us a commit).